mods/ts/rules/gdi-support.yaml

```diff
@@ -25,6 +25,26 @@ GAWALL:
 	LineBuild:
 		NodeTypes: wall, turret
 
+GAGATE_A:
+	Inherits: ^Gate_A
+	Buildable:
+		Queue: Defense
+		BuildPaletteOrder: 100
+		Prerequisites: gapile, ~structures.gdi
+	Tooltip:
+		Name: GDI Gate
+		Description: Stops infantry and blocks enemy fire.\nCan NOT be crushed by tanks.
```

Comment:
Kind of hard to come up with a description for something as simple as a gate... I suggest changing this to something like "Allows friendly units to pass through walls".